### ISSUE TITLE
Issue with actual develop branch and downloading images from Telegram

### ISSUE THREAD

**Issue by sohamakl** [LukeSkywalker92/TeleFrame#105]
I have problems with the current branch `develop`.  When I start TeleFrame via SSH with `tf.sh` and then send a picture I get the following error: 

![image](https://user-images.githubusercontent.com/8607903/73118856-3fe80c00-3f5a-11ea-90a5-3f731e8750c2.png)

Does anyone have any idea what going wrong?

**Comment by gegu** [comment]
I can't reproduce the problem using your steps.

Is the image displayed in TeleFrame?

The error occurs during download in a node module. `substr` is not used in bot.js.


You can change line 150 in _bot.js_ to additionally log a stack trace:
```patch
@@ -145,11 +145,11 @@ var Bot = class {
                 chatName,
                 ctx.message.message_id
               );
             })
             .catch((err) => {
-              this.logger.error(err);
+              this.logger.error(err.stack);
             });
           }
         })
         .catch((err) => {
           this.logger.error('Download: ' + err.toString());
```

**Comment by sohamakl** [comment]
No, the image is not displayed in TeleFrame. Adding `.stack` in line 150 does not change the error message. The images directory is empty.

By the way, any idea hot to scale images in github's wiki?

**Comment by sohamakl** [comment]
I have found the solution.  I just forgot to switch off the privacy mode for the bot in Telegram `/setprivacy disable`.  Since I use a Telegram group. 

Maybe you can give an additional hint to this option in the error message.

**Comment by gegu** [comment]
It would be nice to find out where the error occurs.

Could you please change line 155 as follows and reproduce the error
```patch
@@ -150,11 +150,11 @@ var Bot = class {
               this.logger.error(err);
             });
           }
         })
         .catch((err) => {
-          this.logger.error('Download: ' + err.toString());
+          this.logger.error('Download: fileId: ' + fileId + '\n' + err.stack);
           ctx.reply('Sorry: ' + err.toString());
         });
     });
 
     this.bot.catch((err) => {
```

**Comment by sohamakl** [comment]
With the change I get the following message:

![image](https://user-images.githubusercontent.com/8607903/73125594-b875b980-3fa8-11ea-89b5-cc68a71ec298.png)

But now I'm sure to know the reason. After I configured the bot in Telegram, I assigned it directly to a group. I then sent the first photo out of the group.

This does not work until you have sent a picture directly to the bot. Afterwards it goes then also over the group.

An alternative could be to start the bot from within the group by command `/start`.
